libs/common/src/platform/enums/encryption-type.enum.ts

```diff
@@ -8,6 +8,24 @@ export enum EncryptionType {
   Rsa2048_OaepSha1_HmacSha256_B64 = 6,
 }
 
+const EncryptionTypeNames = {
+  [EncryptionType.AesCbc256_B64]: "AesCbc256_B64",
+  [EncryptionType.AesCbc128_HmacSha256_B64]: "AesCbc128_HmacSha256_B64",
+  [EncryptionType.AesCbc256_HmacSha256_B64]: "AesCbc256_HmacSha256_B64",
+  [EncryptionType.Rsa2048_OaepSha256_B64]: "Rsa2048_OaeppSha256_B64",
+  [EncryptionType.Rsa2048_OaepSha1_B64]: "Rsa2048_OaepSha1_B64",
+  [EncryptionType.Rsa2048_OaepSha256_HmacSha256_B64]: "Rsa2048_OaepSha256_HmacSha256_B64",
+  [EncryptionType.Rsa2048_OaepSha1_HmacSha256_B64]: "Rsa2048_OaepSha1_HmacSha256_B64",
+};
+
+export function encryptionTypeToString(encryptionType: EncryptionType): string {
+  if (encryptionType in EncryptionTypeNames) {
+    return EncryptionTypeNames[encryptionType];
```

Comment:
suggestion: you could also use the enum directly
```suggestion
    return EncryptionType[encryptionType];
```

Reply:
Ah, you're right! Sometimes I mix up syntax between different languages. Removed the unnecessary enum as well.